templates/torrc-sample

```diff
@@ -10,6 +10,10 @@ HiddenServicePort 80 10.11.0.2:80
 HiddenServiceDir /var/lib/tor/bitcoin-p2p
 HiddenServicePort <bitcoin-p2p-port> 10.11.1.1:<bitcoin-p2p-port>
 
+# Bitcoin Core RPC Hidden Service
```

Comment:
I've been trying to expose Bitcoin RPC on an old RaspiBolt setup I have (for learning purposes, connecting Specter) and got properly stuck trying to access it. 

Came across here to look at how Luke did it and found this little nugget which is exactly what I needed to get things working on my end :heart_eyes: